#
# Sent upstream (jwz) 2009-06-16
#
Index: xscreensaver-git/hacks/Makefile.in
===================================================================
--- xscreensaver-git.orig/hacks/Makefile.in	2016-05-28 19:05:32.237678724 +0200
+++ xscreensaver-git/hacks/Makefile.in	2016-05-28 19:05:32.233678725 +0200
@@ -273,7 +273,7 @@
 
 
 default: all
-all: $(EXES) $(RETIRED_EXES)
+all: $(EXES)
 
 install:   install-program   install-scripts install-xml install-man
 uninstall: uninstall-program uninstall-xml uninstall-man
Index: xscreensaver-git/hacks/glx/Makefile.in
===================================================================
--- xscreensaver-git.orig/hacks/glx/Makefile.in	2016-05-28 19:05:32.237678724 +0200
+++ xscreensaver-git/hacks/glx/Makefile.in	2016-05-28 19:05:32.233678725 +0200
@@ -261,7 +261,7 @@
 
 
 default: all
-all: $(EXES) $(RETIRED_EXES)
+all: $(EXES)
 
 install:   install-program    install-xml   install-man
 uninstall: uninstall-program  uninstall-xml uninstall-man
