#
# Sent upstream (jwz) 2009-06-16
#
Index: xscreensaver/hacks/Makefile.in
===================================================================
--- xscreensaver.orig/hacks/Makefile.in	2009-09-06 20:22:20.000000000 +0200
+++ xscreensaver/hacks/Makefile.in	2009-09-06 20:41:08.000000000 +0200
@@ -254,7 +254,7 @@
 
 
 default: all
-all: $(EXES) $(RETIRED_EXES)
+all: $(EXES)
 
 install:   install-program   install-scripts install-xml install-man
 uninstall: uninstall-program uninstall-xml uninstall-man
Index: xscreensaver/hacks/glx/Makefile.in
===================================================================
--- xscreensaver.orig/hacks/glx/Makefile.in	2009-09-06 20:22:20.000000000 +0200
+++ xscreensaver/hacks/glx/Makefile.in	2009-09-06 20:41:08.000000000 +0200
@@ -218,7 +218,7 @@
 
 
 default: all
-all: $(EXES) $(RETIRED_EXES)
+all: $(EXES)
 
 install:   install-program    install-xml   install-man
 uninstall: uninstall-program  uninstall-xml uninstall-man
