#
# Sent upstream (jwz) 2009-06-16
#
--- a/hacks/Makefile.in
+++ b/hacks/Makefile.in
@@ -269,7 +269,7 @@
 
 
 default: all
-all: $(EXES) $(RETIRED_EXES)
+all: $(EXES)
 
 install:   install-program   install-scripts install-xml install-man
 uninstall: uninstall-program uninstall-xml uninstall-man
--- a/hacks/glx/Makefile.in
+++ b/hacks/glx/Makefile.in
@@ -283,7 +283,7 @@
 
 
 default: all
-all: $(EXES) $(RETIRED_EXES)
+all: $(EXES)
 
 install:   install-program    install-xml   install-man
 uninstall: uninstall-program  uninstall-xml uninstall-man
