#
# Sent upstream (jwz) 2013-11-23
#

Index: xscreensaver/driver/Makefile.in
===================================================================
--- xscreensaver.orig/driver/Makefile.in	2013-11-23 16:17:34.000000000 +0100
+++ xscreensaver/driver/Makefile.in	2013-11-23 18:10:35.000000000 +0100
@@ -736,30 +736,30 @@
 
 # How we build object files in this directory.
 .c.o:
-	$(CC) -c $(INCLUDES) $(DEFS) $(CFLAGS) $(X_CFLAGS) $<
+	$(CC) -c $(INCLUDES) $(DEFS) $(CPPFLAGS) $(CFLAGS) $(X_CFLAGS) $<
 
 .m.o:
-	$(OBJCC) -c $(INCLUDES) $(DEFS) $(CFLAGS) $(X_CFLAGS) $<
+	$(OBJCC) -c $(INCLUDES) $(DEFS) $(CPPFLAGS) $(CFLAGS) $(X_CFLAGS) $<
 
 # subprocs takes an extra -D option.
 subprocs.o: subprocs.c
-	$(CC) -c $(INCLUDES) $(SUBP_DEFS) $(CFLAGS) $(X_CFLAGS) \
+	$(CC) -c $(INCLUDES) $(SUBP_DEFS) $(CPPFLAGS) $(CFLAGS) $(X_CFLAGS) \
 	  $(srcdir)/subprocs.c
 
 # xscreensaver takes an extra -D option.
 xscreensaver.o: xscreensaver.c
-	$(CC) -c $(INCLUDES) $(DEFS) $(INTL_DEFS) $(CFLAGS) $(X_CFLAGS) \
+	$(CC) -c $(INCLUDES) $(DEFS) $(INTL_DEFS) $(CPPFLAGS) $(CFLAGS) $(X_CFLAGS) \
 	  $(srcdir)/xscreensaver.c
 
 # demo-Gtk takes extra -D options, and an extra -I option.
 demo-Gtk.o: demo-Gtk.c
 	$(CC) -c $(INCLUDES) $(SUBP_DEFS) -I$(ICON_SRC) \
-	  $(GTK_DEFS) $(INTL_DEFS) $(CFLAGS) $(X_CFLAGS) \
+	  $(GTK_DEFS) $(INTL_DEFS) $(CPPFLAGS) $(CFLAGS) $(X_CFLAGS) \
 	  $(srcdir)/demo-Gtk.c
 
 # demo-Gtk-conf takes an extra -D option.
 demo-Gtk-conf.o: demo-Gtk-conf.c
-	$(CC) -c $(INCLUDES) $(CONF_DEFS) $(GTK_DEFS) $(CFLAGS) $(X_CFLAGS) \
+	$(CC) -c $(INCLUDES) $(CONF_DEFS) $(GTK_DEFS) $(CPPFLAGS) $(CFLAGS) $(X_CFLAGS) \
 	  $(srcdir)/demo-Gtk-conf.c
 
 
@@ -862,7 +862,7 @@
 
 
 xdpyinfo.o: xdpyinfo.c
-	$(CC) -c $(INCLUDES) -DHAVE_GLX $(CFLAGS) $(X_CFLAGS) \
+	$(CC) -c $(INCLUDES) -DHAVE_GLX $(CPPFLAGS) $(CFLAGS) $(X_CFLAGS) \
 	  $(srcdir)/xdpyinfo.c
 
 xdpyinfo: xdpyinfo.o
Index: xscreensaver/hacks/Makefile.in
===================================================================
--- xscreensaver.orig/hacks/Makefile.in	2013-11-23 18:07:11.000000000 +0100
+++ xscreensaver/hacks/Makefile.in	2013-11-23 18:11:08.000000000 +0100
@@ -521,7 +521,7 @@
 
 # How we build object files in this directory.
 .c.o:
-	$(CC) -c $(INCLUDES) $(DEFS) $(CFLAGS) $(X_CFLAGS) $<
+	$(CC) -c $(INCLUDES) $(DEFS) $(CPPFLAGS) $(CFLAGS) $(X_CFLAGS) $<
 
 
 # Make sure these are regenerated when the version number ticks.
Index: xscreensaver/hacks/glx/Makefile.in
===================================================================
--- xscreensaver.orig/hacks/glx/Makefile.in	2013-11-23 18:07:11.000000000 +0100
+++ xscreensaver/hacks/glx/Makefile.in	2013-11-23 18:12:44.000000000 +0100
@@ -476,7 +476,7 @@
 
 # How we build object files in this directory.
 .c.o:
-	$(CC) -c $(INCLUDES) $(DEFS) $(CFLAGS) $(X_CFLAGS) $<
+	$(CC) -c $(INCLUDES) $(DEFS) $(CPPFLAGS) $(CFLAGS) $(X_CFLAGS) $<
 
 
 # How to build the "xscreensaver-gl-helper" program, that lets the daemon
@@ -491,7 +491,7 @@
 # These hacks use a slightly-differently-compiled variant of xlockmore.c.
 # This is how to make the the other .o file from it.
 #
-XLM_CFLAGS=-DUSE_GL $(INCLUDES) $(DEFS) $(CFLAGS) $(X_CFLAGS)
+XLM_CFLAGS=-DUSE_GL $(INCLUDES) $(DEFS) $(CPPFLAGS) $(CFLAGS) $(X_CFLAGS)
 xlockmore-gl.o: $(HACK_SRC)/xlockmore.c
 	$(CC) -o $@ -c $(XLM_CFLAGS) $(HACK_SRC)/xlockmore.c
 
Index: xscreensaver/utils/Makefile.in
===================================================================
--- xscreensaver.orig/utils/Makefile.in	2012-07-28 18:55:08.000000000 +0200
+++ xscreensaver/utils/Makefile.in	2013-11-23 18:13:04.000000000 +0100
@@ -185,7 +185,7 @@
 
 # How we build object files in this directory.
 .c.o:
-	$(CC) -c $(INCLUDES) $(DEFS) $(CFLAGS) $(X_CFLAGS) $<
+	$(CC) -c $(INCLUDES) $(DEFS) $(CPPFLAGS) $(CFLAGS) $(X_CFLAGS) $<
 
 
 # Rules for generating the VMS makefiles on Unix, so that it doesn't have to
