#
# Sent upstream (jwz) 2013-07-24
#
Index: xscreensaver/hacks/glx/quasicrystal.c
===================================================================
--- xscreensaver.orig/hacks/glx/quasicrystal.c	2013-07-16 23:42:13.000000000 +0200
+++ xscreensaver/hacks/glx/quasicrystal.c	2013-07-24 20:37:45.000000000 +0200
@@ -179,7 +179,9 @@
   if (! wire)
     {
       unsigned char *o;
-      tex_width = 4096;
+
+      glGetIntegerv(GL_MAX_TEXTURE_SIZE, &tex_width);
+      if (tex_width > 4096) tex_width = 4096;
       tex_data = (unsigned char *) calloc (4, tex_width);
       o = tex_data;
       for (i = 0; i < tex_width; i++)
