Description: Fix dot eating segfault in pacman_level.c
Author: Tormod Volden <debian.tormod@gmail.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1196251
Forwarded: jwz 2013-07-01

Index: xscreensaver/hacks/pacman_level.c
===================================================================
--- xscreensaver.orig/hacks/pacman_level.c	2012-07-28 18:55:07.000000000 +0200
+++ xscreensaver/hacks/pacman_level.c	2013-07-01 19:37:31.000000000 +0200
@@ -539,7 +539,7 @@
 {
     int ret = False;
     int i;
-    for (i = 0; i <= NUM_BONUS_DOTS; i++) {
+    for (i = 0; i < NUM_BONUS_DOTS; i++) {
 /*     fprintf(stderr,"is bonus: passed x (%d, %d) bonus (%d, %d)\n",x,y,bonus_dots[i].x, bonus_dots[i].y); */
         if (x == pp->bonus_dots[i].x && y == pp->bonus_dots[i].y) {
             ret = True;
@@ -553,7 +553,7 @@
 static void
 check_bonus_idx (int idx)
 {
-    assert (0 <= idx && idx <= NUM_BONUS_DOTS);
+    assert (0 <= idx && idx < NUM_BONUS_DOTS);
 }
 
 int
