#
# Received from upstream (jwz) 2013-08-24
#
Index: xscreensaver/hacks/hexadrop.c
===================================================================
--- xscreensaver.orig/hacks/hexadrop.c	2013-07-16 23:42:13.000000000 +0200
+++ xscreensaver/hacks/hexadrop.c	2013-08-25 09:43:34.000000000 +0200
@@ -359,8 +359,21 @@
 hexadrop_event (Display *dpy, Window window, void *closure, XEvent *event)
 {
   state *st = (state *) closure;
+  Bool bonkp = False;
 
-  if (event->type == ButtonPress || event->type == KeyPress)
+
+  if (event->type == ButtonPress)
+    bonkp = True;
+  else if (event->type == KeyPress)
+    {
+      KeySym keysym;
+      char c = 0;
+      XLookupString (&event->xkey, &c, 1, &keysym, 0);
+      if (c == ' ' || c == '\t' || c == '\r' || c == '\n')
+        bonkp = True;
+    }
+
+  if (bonkp)
     {
       cell *c = st->cells;
       st->cells = 0;
