#
# Sent upstream (jwz) 2013-10-18
#
Index: xscreensaver/driver/xscreensaver-command.c
===================================================================
--- xscreensaver.orig/driver/xscreensaver-command.c	2013-02-07 21:08:39.000000000 +0100
+++ xscreensaver/driver/xscreensaver-command.c	2013-10-18 22:39:51.000000000 +0200
@@ -321,10 +321,10 @@
       exit (i);
     }
 
-  if (*cmd == XA_ACTIVATE || *cmd == XA_LOCK ||
-      *cmd == XA_NEXT || *cmd == XA_PREV || *cmd == XA_SELECT)
+  if ((*cmd == XA_ACTIVATE || *cmd == XA_LOCK ||
+      *cmd == XA_NEXT || *cmd == XA_PREV || *cmd == XA_SELECT) && isatty(0))
     /* People never guess that KeyRelease deactivates the screen saver too,
-       so if we're issuing an activation command, wait a second. */
+       so if we're issuing an activation command on a command line, wait a second. */
     sleep (1);
 
   i = xscreensaver_command (dpy, *cmd, arg, True, NULL);
