#
# "Branding", show Planet Debian posts instead of Wikipedia pages
#
--- a/driver/XScreenSaver.ad.in
+++ b/driver/XScreenSaver.ad.in
@@ -59,11 +59,11 @@
 *ignoreUninstalledPrograms: False
 *authWarningSlack:	20
 
-*textMode:		file
+*textMode:		url
 *textLiteral:		XScreenSaver
 *textFile:		@DEFAULT_TEXT_FILE@
 *textProgram:		fortune
-*textURL:		https://en.wikipedia.org/w/index.php?title=Special:NewPages&feed=rss
+*textURL:		https://planet.debian.org/rss20.xml
 
 ! When a saver writes an error message to stdout/stderr, it can be printed
 ! on the screen.
