Index: xscreensaver/driver/screensaver-properties.desktop.in
===================================================================
--- xscreensaver.orig/driver/screensaver-properties.desktop.in	2010-04-16 22:55:00.000000000 +0200
+++ xscreensaver/driver/screensaver-properties.desktop.in	2010-09-11 22:20:08.000000000 +0200
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Exec=xscreensaver-demo
-Icon=xscreensaver
+Icon=xscreensaver.svg
 Terminal=false
 _Name=Screensaver
 _Comment=Change screensaver properties
