Index: xscreensaver/driver/XScreenSaver.ad.in
===================================================================
--- xscreensaver.orig/driver/XScreenSaver.ad.in	2012-07-28 18:16:44.000000000 +0200
+++ xscreensaver/driver/XScreenSaver.ad.in	2012-07-28 18:18:39.000000000 +0200
@@ -58,11 +58,11 @@
 *captureStderr: 	True
 *ignoreUninstalledPrograms: False
 
-*textMode:		file
+*textMode:		url
 *textLiteral:		XScreenSaver
 *textFile:		@DEFAULT_TEXT_FILE@
 *textProgram:		fortune
-*textURL:		http://twitter.com/statuses/public_timeline.atom
+*textURL:		http://planet.debian.org/rss20.xml
 !*textURL:		http://www.livejournal.com/stats/latest-rss.bml
 
 *overlayTextForeground:	#FFFF00
