Index: xscreensaver/driver/XScreenSaver.ad.in
===================================================================
--- xscreensaver.orig/driver/XScreenSaver.ad.in	2013-02-07 21:17:04.000000000 +0100
+++ xscreensaver/driver/XScreenSaver.ad.in	2013-02-07 21:17:04.000000000 +0100
@@ -58,11 +58,11 @@
 *captureStderr: 	True
 *ignoreUninstalledPrograms: False
 
-*textMode:		file
+*textMode:		url
 *textLiteral:		XScreenSaver
 *textFile:		@DEFAULT_TEXT_FILE@
 *textProgram:		fortune
-*textURL:		http://en.wikipedia.org/w/index.php?title=Special:NewPages&feed=rss
+*textURL:		http://planet.debian.org/rss20.xml
 
 *overlayTextForeground:	#FFFF00
 *overlayTextBackground:	#000000
