#
# Prompts more consistent with the Debian experience (?)
#
--- a/driver/XScreenSaver.ad.in
+++ b/driver/XScreenSaver.ad.in
@@ -327,10 +327,11 @@
 ! Static text in the dialog boxes:
 !
 *passwd.heading.label:		XScreenSaver %s
-*passwd.body.label:		This screen is locked.
+*passwd.body.label:		Please enter your password.
 *passwd.unlock.label:		OK
 *passwd.login.label:		New Login
 *passwd.user.label:		Username:
+*passwd.passwd.label:		Password:
 
 *splash.heading.label:		XScreenSaver %s
 *splash.body.label:		Copyright \251 1991-2019 by
