#
# Prompts more consistent with the Debian experience (?)
#
Index: xscreensaver-git/driver/XScreenSaver.ad.in
===================================================================
--- xscreensaver-git.orig/driver/XScreenSaver.ad.in	2016-05-28 19:06:03.673677107 +0200
+++ xscreensaver-git/driver/XScreenSaver.ad.in	2016-05-28 19:06:03.669677107 +0200
@@ -435,10 +435,11 @@
 *Dialog.shadowThickness:	2
 
 *passwd.heading.label:		XScreenSaver %s
-*passwd.body.label:		This screen is locked.
+*passwd.body.label:		Please enter your password.
 *passwd.unlock.label:		OK
 *passwd.login.label:		New Login
 *passwd.user.label:		Username:
+*passwd.passwd.label:		Password:
 *passwd.thermometer.width:	8
 *passwd.asterisks:              True
 *passwd.uname:                  True
