Index: xscreensaver/driver/XScreenSaver.ad.in
===================================================================
--- xscreensaver.orig/driver/XScreenSaver.ad.in	2013-11-23 16:16:48.000000000 +0100
+++ xscreensaver/driver/XScreenSaver.ad.in	2013-11-23 16:16:48.000000000 +0100
@@ -420,10 +420,11 @@
 *Dialog.shadowThickness:	2
 
 *passwd.heading.label:		XScreenSaver %s
-*passwd.body.label:		This screen is locked.
+*passwd.body.label:		Please enter your password.
 *passwd.unlock.label:		OK
 *passwd.login.label:		New Login
 *passwd.user.label:		Username:
+*passwd.passwd.label:		Password:
 *passwd.thermometer.width:	8
 *passwd.asterisks:              True
 *passwd.uname:                  True
